mxv/templates/mxv/navbar.html

```diff
@@ -1,24 +1,12 @@
 {% load static %}
 <nav class="navbar navbar-expand-lg navbar-light bg-light">
     <div class="container">
-    <a class="navbar-brand" href="{% url 'index' %}"><img src="{% static "mxv/images/logo.png" %}" alt="Logo" style="width: 20rem;"/></a>
+    <a class="navbar-brand" href="{% url 'index' %}"><img src="{% static "mxv/images/logo.png" %}" alt="Logo" style="max-width: 300px;"/></a>
     <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
         <span class="navbar-toggler-icon"></span>
     </button>
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
```

Comment:
There is nothing left inside this div; maybe get rid of it entirely?

Reply:
have removed the empty ul